RitoxicGame/space-trader
# updated travel ui to use new assets

## What did you add in this branch?

Fixed text, script, and layouts so that the Travel Confirmation popup uses the new UI asset made by Gil.
Travel UI should also retain shape and layout through aspect ratio/resolution changes.

## How can we test these changes?

Go into Game. Attempt to travel to a planet.
Does the popup window appear properly?
Do the buttons give decent feedback on click and on hover?
Does it look nice?
Fiddle with the resolution: does the popup ever become hopelessly deformed?

Note: there is a known(?) issue where the first planet click of the game *not* bring up the travel popup. Simply click again, and the popup will begin appearing as normal.
## Relevant screenshots (if applicable):


## What issue is this PR associated with?

https://github.com/RitoxicGame/space-trader/issues/10#issue-2211958828
## What week is this due by?

Final Build/**today**
